pennylane/gradients/parameter_shift_cv.py

```diff
@@ -493,6 +494,7 @@ def param_shift_cv(
     fallback_fn=finite_diff,
     f0=None,
     force_order2=False,
+    shots=None,

```

Comment:
The CV device does not inherit from _qubit_device, so I think more changes are to be made separately in order to integrate CV variables.

Reply:
I added this change because the `vjp` function passes the `shots` argument to the `gradient_fn`, which is necessary to compute the gradients correctly in the shot vector cases. But doing this breaks some tests because `parameter_shift_cv` does not have a `shots` argument. I don't think we want to check if the `gradient_fn` is for a qubit device or a CV device because it breaks abstraction.

What do you think we should do in this case?

Reply:
Oh okay I understand better, maybe add a TODO next to the disable.